### ISSUE TITLE
auto deploy failing

### ISSUE THREAD

**Issue by alloy** [relay-tools/relay-compiler-language-typescript#167]
It looks like something changed somewhere that’s currently breaking deploys, but unsure if it’s `auto` or the `@artsy` config repo.

@zephraph Any idea?

```
321●  note      @artsy found: undefined
322●  note      @artsy/auto-config/package.json found: [object Object]
323✖  error     You're using a deprecated configuration option!
324
325The "labels" option no longer supports configuration with an object.
326Instead supply your labels as an array of label objects.
327
328ex:
329
330|  {
331|    "labels": [
332|      {
333|        "name": "my-label",
334|        "description": "Really big stuff",
335|        "type": "major"
336|      }
337|    ]
338|  }
339Script failed with status 1
```

**Comment by zephraph** [comment]
Yep! I’ll fix it, give me a minute.

**Comment by alloy** [comment]
:rocket: Issue was released in v10.1.4 :rocket:
